visualization/app/codeCharta/ui/customConfigs/customConfigs.component.ts

```diff
@@ -76,6 +76,7 @@ export class CustomConfigsController implements FilesSelectionSubscriber {
 
 	removeCustomConfig(configId) {
 		CustomConfigHelper.deleteCustomConfig(configId)
+		this.loadCustomConfigs()
```

Comment:
In a perform manner I would remove the deleted Config from `this._viewModel.dropDownCustomConfigItemGroups` instead of accessing the localStorage API which to avoid I/O.
But this will do the job and I think it is not performance critical. So, I am fine with it :-)